framework/tests/all-systems/java21/JEP441.java

```diff
@@ -8,7 +8,7 @@
 // These are examples copied from:
 // https://openjdk.org/jeps/441
 
-@SuppressWarnings("i18n") // true postives.
+@SuppressWarnings({"i18n", "nullness:nulltest.redundant"}) // true postives.
```

Comment:
How about adding `@Nullable` annotations to the corresponding method parameters? That would make these additional valid tests for the new feature.

Reply:
> How about adding `@Nullable` annotations to the corresponding method parameters? That would make these additional valid tests for the new feature.

Thanks! addressed!